docs/content/concepts/assets/asset-observations.mdx

```diff
@@ -117,7 +117,16 @@ staleness calculations for downstream assets.
 The <PyObject object="observable_source_asset" /> decorator provides a convenient way to define source assets with observation functions. The below observable source asset takes a file hash and returns it as the logical version. Every time you run the observation function, a new observation will be generated with this hash set as logical version.
 
 ```python file=/concepts/assets/observable_source_assets.py startafter=start endbefore=end
```

Comment:
nit: could we make the markers and the comments more self explanatory, something like "*_start_marker" and "*_end_marker". `#start` and `#end` in code may confuse people in case they are just reading the code snippets in examples dir

Reply:
done